examples/complete/dns_records.txt

```diff
@@ -0,0 +1,2 @@
+;; AAAA Records
+test-ar.aaaa.example.dev.cloud.ibm.com.	900	IN	AAAA	2001::4
```

Comment:
Can we have 2-3 records ? that looks more real scenario.

Reply:
Done